README.md

```diff
@@ -171,7 +171,7 @@ make graph-properties-convert
 Many Galois/Lonestar applications work with graphs. We store graphs in a binary format
 called *galois graph file* 
 (`.gr` file extension). Other formats such as edge-list or Matrix-Market can be
-converted to `.gr` format with `graph-convert` tool provided in galois. 
+converted to `.gr` format with `graph-convert` tool provided in katana. 
```

Comment:
I'd avoid this phrasing altogether and simply reference the graph convert tool itself. Anyway, all this conversion documentation and tooling is out of date because `.gr` files are not directly supported anymore.